templates/spec/solidus_starter_frontend_helper.rb

```diff
@@ -56,4 +56,6 @@
 
     ActionController::Base.cache_store = original_cache_store
   end
+
+  config.include SolidusStarterFrontend::TestingSupport::ControllerRequests, type: :controller
```

Comment:
This is needed for the controller specs to find the application routes. Otherwise, the controller specs would get the routes from the Spree engine through the following call sequence:

* `require 'solidus_dev_support/rspec/feature_helper'` (`spec/solidus_starter_frontend_helper.rb`)
* `require 'solidus_dev_support/rspec/rails_helper'` (`solidusio/solidus_dev_support/lib/solidus_dev_support/rspec/feature_helper.rb`)
* `config.include Spree::TestingSupport::ControllerRequests, type: :controller` (`solidusio/solidus_dev_support/lib/solidus_dev_support/rspec/rails_helper.rb`)

Reply:
I haven't tried namespacing each of the URL helper calls. However, I think it should be okay to override the routes in every spec file, like this:

```ruby
RSpec.describe CheckoutController, type: :controller do
  routes { Rails.application.routes }
end
```

Would you prefer this approach over overriding all of the controller specs only once in `solidus_starter_frontend_helper.rb`?

Reply:
@elia Okay, let's go with inlining `solidus_dev_support/rspec/feature_helper` and removing the `config.include Spree::TestingSupport::ControllerRequests, type: :controller` line. TODO.